be/src/util/mem_info.cpp

```diff
@@ -97,6 +98,8 @@ void MemInfo::process_cache_gc(int64_t& freed_mem) {
     freed_mem +=
             StoragePageCache::instance()->get_page_cache_mem_consumption(segment_v2::DATA_PAGE);
     StoragePageCache::instance()->prune(segment_v2::DATA_PAGE);
+    freed_mem += SegmentLoader::instance()->segment_cache_mem_consumption();
+    SegmentLoader::instance()->prune_all();
```

Comment:
It's too aggressive, we may just need to call `prune_all()` when full gc.
Segment cache is quite useful on accelerating queries and MoW index lookup.

Reply:
fixed,
minor gc only prune outdated segment cache
full gc prune all